.gitignore

```diff
@@ -2,4 +2,4 @@
 /tests/cucumber/node_modules/
 .vagrant/*
 autogenerated_by_travis.sh
-
+FAILURE
```

Comment:
The continuous-integration shell script creates that as an empty file if any single test fails along the way. At the end of the test run if that empty file is present, then the scripts returns with a non-zero exit code. This causes the build in any CI server return as a failure.